### ISSUE TITLE
DS3 does not funtion after H-Encore update to 2.0 (Possible Auto-launch bug)

### ISSUE THREAD

**Issue by yodafire** [TheOfficialFloW/h-encore#74]

**Comment by yodafire** [comment]
Hi, its me again, okay regarding the DS3 controller, I used and inquired about your "minivitatv" plugin, it was fixed and we collaborated happily :).

However, I decided to update my H-Encore today to the new version 2.0 that you have released, the quick installation method was a no go, it kept giving me an error for copying the file over to the location in vita shell (0x8.... sorry could not remember).

Anyhow, when H-Encore is launched without the bootstrap menu, the controller pairs up fine, but there is no joystick moment, no button press being acknowledged, the only button that is able to work is the PS button on the DS3 which allows me to access the menu on the vita. nothing else functions, so i checked and lunched H-Encore again, (no bootstrap), same thing happens. 

But when I thought about this, I tried to do it the old way, so restarted the vita, and launched H-Encore whilst I had the R trigger pressed down, enabling me to see the bootstrap menu, I exit as usual, and then I  pair the controller (PS button), and all of a sudden the controller functions as usual. 

I did this 4 times, and can clearly see there is an actual problem persisting with the way version 2.0 H-Encore boots into the Vita, and when trying to connect the DS3 controller, its ceases to function.

This could be a potentially minor bug, but I thought I would raise this concern to you, so it will avoid others to post up the same thing and avoiding them bugging you afterwards.

Thanks 

Yoda
